### PR TITLE
Update requirements to list MySQL 9.0 support

### PR DESCRIPTION
**Pull request by Gaitholabi**

### CODE REVIEW

**Review by AydinHassan — APPROVED** [review]

File: guides/installation/requirements.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ You can use these commands to check your actual environment:
 
 * MySQL
 
-  * Recommended version: 8.0
+  * Compatible versions: 8.0 and 9.0
```
Comment (shyim) [inline guides/installation/requirements.md:58]: MySQL 9 is an innovation release. I would add something like not recommended to use in production or even tbh not mention here 🤔

Comment (Gaitholabi) [reply guides/installation/requirements.md:58]: I tend to agree, maybe we could ignore listing it here, and if someone asks specifically, we could tell them that it should be fine but not recommended

Comment (sushmangupta) [reply guides/installation/requirements.md:58]: @Gaitholabi or @shyim , shall I close this PR or change "recommended version to compatible version" ?

Comment (Gaitholabi) [reply guides/installation/requirements.md:58]: @sushmangupta i'll close it for now 👍

**Pull request closed without merging**